server/chat-plugins/trivia.js

```diff
@@ -923,7 +923,7 @@ class TimerModeTrivia extends Trivia {
 		}
 
 		let rowAdded = false;
-		innerBuffer.forEach((players, pointValue) => {
+		for (let [pointValue, players] of innerBuffer) {
 			if (!players.length) return false;
```

Comment:
```suggestion
			if (!players.length) continue;
```

`return` becomes `continue` when refactoring `.forEach` to `for...of`